src/webui/webapplication.h

```diff
@@ -157,4 +157,13 @@ class WebApplication final
     bool m_isHostHeaderValidationEnabled;
     bool m_isHttpsEnabled;
     QString m_contentSecurityPolicy;
+
+    // Custom header
+    struct CustomHeader
+    {
+        QString header;
```

Comment:
header -> name

Reply:
what about `headerName`?

Reply:
Isn't "name" field of "CustomHeader" struct clear enough? Otherwise you should have "headerValue" as well... but they both are redundant.